src/cluster_legacy.c

```diff
@@ -6582,14 +6578,29 @@ int clusterAsmOnEvent(slotRangeArray *slot_ranges, int state, void *arg) {
             clusterSaveConfigOrDie(1);
             clusterAsmRequest(slot_ranges, ASM_REQUEST_CONFIG_UPDATED, NULL, NULL);
             break;
+        case ASM_EVENT_IMPORT_FINALIZED:
+            serverLog(LL_NOTICE, "Import operation finalized for slot ranges: %s", str);
+            break;
         case ASM_EVENT_MIGRATE_STARTED:
-            serverLog(LL_NOTICE, "Migration started for slot ranges: %s", str);
+            serverLog(LL_NOTICE, "Migrate operation started for slot ranges: %s", str);
             break;
         case ASM_EVENT_MIGRATE_FAILED:
-            serverLog(LL_NOTICE, "Migration failed for slot ranges: %s", str);
+            serverLog(LL_NOTICE, "Migration operation failed for slot ranges: %s", str);
```

Comment:
Align with other logs？
```suggestion
            serverLog(LL_NOTICE, "Migrate operation failed for slot ranges: %s", str);
```